AirCasting/Graph/Graph.swift

```diff
@@ -48,7 +50,7 @@ class UI_PollutionGraph: UIView {
         lineChartView.xAxis.drawLabelsEnabled = true
         lineChartView.xAxis.labelCount = 2
         lineChartView.extraBottomOffset = 25
-        
+                
```

Comment:
try always to keep the changes as clean as possible. this space can lead to unnecessary conflicts for others. Before pushing usually look thru Your changed and revoke committing white space in places where the cade was not touched.